crsqlite_extension/src/crsqlite_extension.c

```diff
@@ -0,0 +1,57 @@
+/* Main file that loads the crsqlite extension
+  Lets try to keep it clean*/
+#include <stdio.h>
+#include <stdlib.h>
+
+#include <sqlite3ext.h>
+
+#include "vtab.h"
+SQLITE_EXTENSION_INIT1
+
+
+
+/* Function to initialize crsqlite Extension, creates metatables if they dont exist */
+int init_crsqlite(sqlite3 *db)
+{
+  //TODO
```

Comment:
The original implementation had version and site id metatables.

We should move this information into function extensions and calculate that version at db startup. 

When I did some perf tests, hitting the version and site id tables were the main bottlenecks and saving them as variables, returned by a function extension, put crsqlite perf on par with regular sqlite.

Reply:
Do they actually need to be accessible from SQL statements? we might get away with having them as part of the vtable struct stored in memory? thats a TODO though.

Reply:
You're right, they don't need to be accessible from SQL statements. Should work from the vtable extension directly 👍